package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,340 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
+=====================================================================
+
+:Authors: John Detlefs
+:Year: 2016
+:Copyright: GNU Public License v3
+
+This module contains the linear dimension reduction method Principal
+Component Analysis. This module constructs a covariance matrix wherein each
+element of the matrix is denoted by (i,j) row-column coordinates. The (i,j)
+coordinate reflects the influence of the of the ith frame's coordinates on the
+jth frame's coordinates of a given trajectory. The eponymous components are the
+eigenvectors of this matrix.
+
```

Comment:
```
This module contains the linear dimensions reduction method Principal Component Analysis (PCA). The PCA 
sorts a simulation into 3N directions of descending variance, with N being the number of atoms. The directions 
are called principle components. The dimensions to be analyzed are reduced by only looking at a few projections 
of the first pca_components. A PCA can be calculated as follows.

# here show a code block to generate a PCA

Now we can see how each principle component adds to the accumulated variance.

# show code block

To reduce the dimensions of the simulation we will transform it into the subspace that explains 80% of the variance. 

# show reduction code block

# below here can come some more technical descriptions
```

This docs for this module are a bit to technical. There is no good clear sentence/paragraph missing what the benefit of using PCA are don't mention by name when you actually do the dimension reduction. I've written above a short piece that does a short general introduction to PCA how it can be used and it explicitaly mentions when I do the dimension reduction. It isn't ideal but it goes straigt to the point without talking to much about the implementation details of the algorithm. It is ok to do that later in the docs but first and foremost we should make it clear what the benefit of the analysis is and how to perform it.